src/Microsoft.AspNetCore.Sockets.Http/ConnectionManager.cs

```diff
@@ -128,33 +134,34 @@ public void Scan()
                 {
                     var status = DefaultConnectionContext.ConnectionStatus.Inactive;
                     var lastSeenUtc = DateTimeOffset.UtcNow;
+                    var connection = c.Value.Connection;
```

Comment:
Destructure in the for loop! `foreach (var (connection, _) in _connections)`

Reply:
....